src/main/scala/gestalt/Quote.scala

```diff
@@ -11,6 +11,7 @@ abstract class Quote(val t: Toolbox, val toolboxName: String) {
   type Quasi = m.internal.ast.Quasi
 
   // fields
+  // args: List[t.Tree[Any]]
```

Comment:
I wanted to emphasize that args can contain trees of `Any` value : `Int`s, `Boolean`s, other `AnyVal`s, `String`s and even Trees. Maybe `List[t.Tree[Any| t.Tree[_]]]` is better?